Source/WebCore/Modules/mediastream/gstreamer/GStreamerStatsCollector.cpp

```diff
@@ -257,8 +257,8 @@ RTCStatsReport::IceCandidateStats::IceCandidateStats(GstWebRTCStatsType statsTyp
 
 RTCStatsReport::IceCandidatePairStats::IceCandidatePairStats(const GstStructure* structure)
     : Stats(Type::CandidatePair, structure)
-    , localCandidateId(gstStructureGetString(structure, "local-candidate-id"_s).toString())
-    , remoteCandidateId(gstStructureGetString(structure, "remote-candidate-id"_s).toString())
+    , localCandidateId(String(gstStructureGetString(structure, "local-candidate-id"_s).span()))
+    , remoteCandidateId(String(gstStructureGetString(structure, "remote-candidate-id"_s).span()))
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?